### ISSUE TITLE
suggestion: resize multiselect-edit-boxes via Admin Panel

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#2297]
#### Issue

This is an enhancement suggestion:

The width and height of multiselect boxes in edit view and both select and multiselect boxes in advanced search filter view is often too small to use it efficiently. Changing this is more trouble than it should be imho.
#### Expected Behavior

Ability to change the max width and length of such multiselect boxes (whether edit or search filter) in Admin panel. SuiteCRM would increase their size until the indicated max width/length is reached...

-> or any other improvement idea. e.g. ability to directly set in studio for each field whether to adapt to the amount of items in the list or not. Or to directly set the width/size of a field in studio.
#### Actual Behavior

multiselect boxes (edit) and select/multiselect boxes (search filter) are quite small and don't adapt to browser window size or amount of items in the list
#### Steps to Reproduce

1.
2.
3.
4.
#### Your Environment

- SuiteCRM Version used: 7.7.4
- Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):
- Environment name and version (e.g. MySQL, PHP 7):
- Operating System and version (e.g Ubuntu 16.04):

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [suggestion: resize multiselect-edit-boxes via Admin Panel](https://trello.com/c/z6v0UR1u/73-suggestion-resize-multiselect-edit-boxes-via-admin-panel)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
